addon/components/polaris-color-picker/alpha-picker.js

```diff
@@ -10,13 +10,13 @@ const VERTICAL_PADDING = 13;
 
 function offsetForAlpha(alpha, sliderHeight, draggerHeight) {
   const slidableArea = sliderHeight - (draggerHeight + VERTICAL_PADDING);
-  return clamp(((1 - alpha) * slidableArea) + VERTICAL_PADDING, 0, sliderHeight - draggerHeight);
+  return clamp((1 - alpha) * slidableArea + VERTICAL_PADDING, 0, sliderHeight - draggerHeight);
```

Comment:
These were there to match the Polaris source code to make updates easier in the future 😳

Reply:
Interestingly polaris started using prettier and they have the [exact same expression](https://github.com/Shopify/polaris/blob/master/src/components/ColorPicker/components/AlphaPicker/AlphaPicker.tsx#L96) now 🙂